src/session/session_api.c

```diff
@@ -13,6 +13,39 @@ static int __session_snapshot(WT_SESSION *, const char *);
 static int __session_rollback_transaction(WT_SESSION *, const char *);
 
 /*
+ * __wt_session_notsup_config --
+ *	Unsupported session actions that have a signature of a config string.
+ */
+int
+__wt_session_notsup_config(WT_SESSION *wt_session, const char *config)
+{
+	WT_SESSION_IMPL *session;
+
+	session = (WT_SESSION_IMPL *)wt_session;
+	WT_UNUSED(config);
+
+	WT_RET_MSG(session, ENOTSUP, "Unsupported session method");
+}
+
+/*
+ * __wt_session_notsup_uri --
+ *	Unsupported session actions that have a signature of a URI and
+ *	a config string.
+ */
+int
+__wt_session_notsup_uri(
+    WT_SESSION *wt_session, const char *uri, const char *config)
+{
+	WT_SESSION_IMPL *session;
+
+	session = (WT_SESSION_IMPL *)wt_session;
+	WT_UNUSED(uri);
+	WT_UNUSED(config);
+
+	WT_RET_MSG(session, ENOTSUP, "Unsupported session method");
+}
+
+/*
```

Comment:
@sueloverso, you don't need two functions, you can do it with a cast (see `cur_std.c` for an example). It doesn't much matter (there are only two after all), FYI if there are ever more.